views/admin/admin-page.php

```diff
@@ -241,6 +241,138 @@
 				<?php endif; ?>
 			</div>
 		</div>
+
+		<div id="benefits-area" class="j-row long-page">
+			<h1 title="<?php esc_attr_e( 'Why connect Jetpack?', 'jetpack' ); ?>"><?php esc_html_e( 'Why connect Jetpack?', 'jetpack' ); ?><sup>*</sup></h1>
+
+			<div class="connect-btn j-col j-sm-12 j-md-12" style="text-align:center">
+				<h2 title="<?php esc_attr_e( 'MORE TRAFFIC TO YOUR SITE', 'jetpack' ); ?>"><?php esc_html_e( 'MORE TRAFFIC TO YOUR SITE', 'jetpack' ); ?></h2>
```

Comment:
@richardmuscat this and other uppercase texts could be added as capitalized lowercase and instead do a `text-transform: uppercase`. This is because some translators might not use uppercase characters and the landing will look inconsistent across different languages.